src/utils/notifications.rs

```diff
@@ -87,7 +87,7 @@ impl<'a> Display for Notification<'a> {
             ),
             SetDefaultBufferSize(size) => write!(
                 f,
-                "defaulting to {} unpack ram",
+                "using up to {} of RAM for unpacking component",
```

Comment:
The settled-upon message was "... for unpacking component**s**" – is that correct, or should it be singular?